x-pack/plugins/fleet/common/openapi/bundled.json

```diff
@@ -4082,6 +4082,9 @@
                 "encrypted_saved_object_encryption_key_required"
               ]
             }
+          },
+          "package_verification_key_id": {
```

Comment:
did you edit this file manually? this is a generated file along with `bundled.yaml`, so you should add the change to the  model yaml files and generated new bundled files as described here: https://github.com/elastic/kibana/tree/main/x-pack/plugins/fleet/common/openapi#tools

Reply:
Ah brill thanks 👍